Source/WebCore/platform/graphics/texmap/TextureMapperPlatformLayerProxy.h

```diff
@@ -53,6 +61,9 @@ class TextureMapperPlatformLayerProxy : public ThreadSafeRefCounted<TextureMappe
     Lock& lock() WTF_RETURNS_LOCK(m_lock) { return m_lock; }
     bool isActive();
 
+    ContentType contentType() { return m_contentType; }
```

Comment:
This may be `const`:

```suggestion
    ContentType contentType() const { return m_contentType; }
```